beacon-chain/rpc/beacon_server.go

```diff
@@ -106,6 +108,18 @@ func (bs *BeaconServer) ForkData(ctx context.Context, _ *ptypes.Empty) (*pbp2p.F
 	return state.Fork, nil
 }
 
+// DomainData fetches the current domain version information from the beacon state.
+func (bs *BeaconServer) DomainData(ctx context.Context, request *pb.DomainRequest) (*pb.DomainResponse, error) {
+	state, err := bs.beaconDB.HeadState(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("could not retrieve beacon state: %v", err)
+	}
+	return &pb.DomainResponse{
+		SignatureDomain:helpers.DomainVersion(state, request.Epoch, params.BeaconConfig().DomainRandao),
```

Comment:
lets do this in another line , then add it to the response

Reply:
fixed, thanks